docs/genrelnotes.py

```diff
@@ -18,8 +18,8 @@
 from glob import glob
 
 relnote_template = '''---
-title: Release %s
-short-description: Release notes for %s
+title: Release {0}
+short-description: Release notes for {1}
```

Comment:
In python 2.7 and 3.0 the requirement for numbers was removed, so this is perfectly valid:
```python
'{} to {}, {}'.format(a, b, c)
```

You can still use the numbers if you want to use a single input multiple times:
```
"""
the new version is {0}, the old was {1}.
isn't {0} great‽
""".format('3.0', '2.6')
```

I'd really prefer to not have the numbers unless we're using a single input twice, like you could do in this case.

Reply:
Yes I feel the same. Once Python 3.5 is deprecated, the `.format(foo)` will likely start changing to `f'... {foo} ...'`

Reply:
Ok removed redundantly place holders.